### PR TITLE
Added plus minus operation with tests

### PR DESCRIPTION
**Pull request by 0matleb2**
Fixes: #18 

### WHAT kind of change does this PR introduce?
This PR changes the interface to add a new button that inverses the sign of the number currently inputted  (+/-). The power was removed from the visible UI. The functionality is still present in the backend, but is not currently accessible to the user. We might add it back if we decide to add another row of operations in a future release. Tests also have been added to cover this new operation.

### HOW is this accomplished?
* flip the sign of the primary number (value being typed).
* Add new UI and non UI tests
* fix multiply and subtract sign in the UI and tests

### Checklist
- [x] Docs have been added / updated to reflect the changes
- [x] I have reviewed and tested the changes :white_check_mark:
- [x] I have squashed my commits to have a reasonable amount of commits

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -159,7 +159,7 @@
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:layout_weight="1"
-            android:text="*"/>
```
Comment (jusleg) [inline app/src/main/res/layout/activity_main.xml]: Good catch!

File: app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt [hunk 2]
```diff
@@ -52,7 +52,7 @@ class MainActivityTest {
         var result  = activity.getResult()
 
         assertEquals("1", result)
-        checkFormula("5-4")
```
Comment (jusleg) [inline app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt]: Pretty crazy that they both look the same in a monospace font. It's better now as it displays the proper minus sign on the calculator

File: app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt [hunk 3]
```diff
@@ -191,6 +190,20 @@ class MainActivityTest {
         checkFormula("15%")
     }
 
+    @Test
+    fun negationTest() {
```
Comment (jusleg) [inline app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt]: Pretty thorough test for negation 👍

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (almiche) [reply app/src/main/res/layout/activity_main.xml]: hey nice one didn't see that one before !

File: app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt (hunk 2 above)
Comment (almiche) [reply app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt]: I agree @jusleg thats quite the point to make!

File: app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt (hunk 3 above)
Comment (almiche) [reply app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt]: wow this puts the fun in fun 💯

**Pull request merged**